src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -21,96 +21,77 @@
 
 <template>
 	<NcAppNavigation :aria-label="t('spreed', 'Conversation list')">
-		<div class="new-conversation"
-			:class="{ 'new-conversation--scrolled-down': !isScrolledToTop }">
-			<SearchBox v-model="searchText"
-				class="conversations-search"
-				:is-searching="isSearching"
-				@input="debounceFetchSearchResults"
-				@submit="onInputEnter"
-				@keydown.enter.native="handleEnter"
+		<div class="new-conversation" :class="{ 'new-conversation--scrolled-down': !isScrolledToTop }">
+			<SearchBox v-model="searchText" class="conversations-search" :is-searching="isSearching"
+				@input="debounceFetchSearchResults" @submit="onInputEnter" @keydown.enter.native="handleEnter"
 				@abort-search="abortSearch" />
 			<NewGroupConversation v-if="canStartConversations" />
 		</div>
+		<div class="button-container">
+			<button class="button" @click="insertValue('is:unread')" :title="t('spreed', 'Unread messages')">{{
+				t('spreed', 'unread') }}</button>
```

Comment:
Translations don't work for you, because it's a new line.
After PR will be merged, all new lines appear here: https://app.transifex.com/nextcloud/nextcloud/spreed/.
Then a community should provide a translation for it (as far as i know, it has to be done manually for each language), and 1-2 times a week, bot will merge new translations into project.